src/pattern-library/components/PlaygroundApp.js

```diff
@@ -39,49 +41,129 @@ export default function PlaygroundApp({
     </>
   );
 
-  const routeGroups = [
-    { title: 'Foundations', routes: getRoutes('foundations') },
-    { title: 'Patterns', routes: getRoutes('patterns') },
-    { title: 'Common Components', routes: getRoutes('components') },
-  ];
+  /**
+   * Header for a primary section of nav
+   *
+   * @param {object} props
+   *   @param {Children} props.children
+   */
+  function NavHeader({ children }) {
+    return (
+      <h2 className="border-b px-2 py-1 font-light text-lg">{children}</h2>
+    );
+  }
 
-  if (extraRoutes.length) {
-    routeGroups.push({ title: 'Application Patterns', routes: extraRoutes });
+  /**
+   * A single navigation link
+   *
+   * @param {object} props
+   *   @param {string} props.title
+   *   @param {PlaygroundRoute['route']} props.route
+   */
+  function NavLink({ title, route }) {
+    return (
+      <li className="-ml-[2px]">
+        {route && (
+          <Link
+            classes={classnames('pl-4 w-full border-l-2 hover:border-l-brand', {
+              'border-l-2 border-brand font-semibold':
+                activeRoute?.route === route,
```

Comment:
The nullability of `activeRoute` made me wonder what the pattern library's 404 page looks like (the case when there is no matching route). It's a very low priority task, but it could use some polish.

Reply:
Noted.